packages/components/src/slot-fill/fill.js

```diff
@@ -69,10 +63,9 @@ function FillComponent( { name, getSlot, children, registerFill, unregisterFill
 
 const Fill = ( props ) => (
 	<Consumer>
-		{ ( { getSlot, registerFill, unregisterFill } ) => (
+		{ ( { registerFill, unregisterFill } ) => (
```

Comment:
We can probably refactor it further in a follow-up PR to avoid using `Consumer` at all and use `useSlot` here as well.

Reply:
yeah, I also thought about adding a `useFills` (similar to `useSlot` and simplify the Slot component implementation using it)